projects/storefrontlib/layout/launch-dialog/config/launch-config.ts

```diff
@@ -99,4 +99,5 @@ export const enum LAUNCH_CALLER {
   SUGGESTED_ADDRESSES = 'SUGGESTED_ADDRESSES',
   COUPON = 'COUPON',
   STOCK_NOTIFICATION = 'STOCK_NOTIFICATION',
+  REORDER = 'REORDER',
```

Comment:
Please move this to the appropriate feature lib and use type augmentation to add the REORDER element in the enum.